buildenv/jenkins/JenkinsfileBase

```diff
@@ -496,6 +500,12 @@ def runTest( ) {
 			}
 			if (!TARGET.startsWith('-f')) {
 				TARGET="_${params.TARGET}"
+			} else if (TARGET.startsWith('-f parallelList.mk') && SPEC.startsWith('zos')) {
```

Comment:
is TARGET order dependent?  as in, will it always startWith '-f parallelList.mk', or could it TARGET.contains() ?